templates/components/itilobject/timeline/form_document_item.html.twig

```diff
@@ -102,6 +102,26 @@
                 <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}" />
                 {{ call_plugin_hook('pre_item_form', {"item": subitem, 'options': params}) }}
 
+                <label id="upload_source_label" class="form-label">{{ __("Upload from") }}</label>
+                <div class="d-inline" role="group" aria-labelledby="upload_source_label">
+                    {% if call('Document::isValidDoc', ['.png']) is not empty %}
+                        <button class="btn btn-sm btn-secondary me-2 d-none" type="button" name="add_screenshot">
+                            <i class="ti ti-photo"></i>
+                            <span>{{ __('Screenshot') }}</span>
+                        </button>
+                    {% endif %}
+                    {% if call('Document::isValidDoc', ['.webm']) is not empty %}
+                        <button class="btn btn-sm btn-secondary me-2 d-none" type="button" name="add_screenrecording">
+                            <i class="ti ti-camera"></i>
+                            <span>{{ __('Screen recording') }}</span>
+                        </button>
+                    {% endif %}
+                </div>
+                <div id="screen_capture_preview" class="w-100">
+                    <div class="previews overflow-x-scroll my-2 d-flex px-2"></div>
+                    <button type="button" name="stop_recording" class="btn btn-secondary d-none">{{ __('Stop recording') }}</button>
+                </div>
+                <hr class="my-1">
```

Comment:
This whole block should probably be skipped when the feature is not available.

When PNG and WebM files are disallowed, or if the `Screenshot.isSupported()` returns `false`, the form title and separator should not be displayed:
![image](https://github.com/glpi-project/glpi/assets/33253653/8cf76ed6-38b2-44f4-a8a1-0ea3b7b474af)